src/util/listQueryBuilder.js

```diff
@@ -451,6 +447,119 @@ async function performAggregationQuery ({
   return paginationMeta
 }
 
+/**
+ * Generic query function to address filters, sort and pagination for stats endpoints
```

Comment:
Do you write `stats` on purpose to include deprecated stats endpoint, or should you replace with `history endpoint`?

Reply:
Good catch, there's a typo. It should be `history endpoint`.